api/swagger.yaml

```diff
@@ -3420,6 +3420,10 @@ definitions:
           > should not be considered stable.
         type: "string"
         example: "7TRN:IPZB:QYBB:VPBQ:UMPP:KARE:6ZNR:XE6T:7EWV:PKF4:ZOJD:TPYS"
+      ContainerdNamespace:
```

Comment:
Should this be a top-level key or something more opaque like a `map[string]string` for runtime config?

Reply:
Hum good question, @thaJeztah ? :stuck_out_tongue: